atlasdb-jdbc/src/main/resources/META-INF/services/io.dropwizard.jackson.Discoverable

```diff
@@ -0,0 +1 @@
+com.palantir.atlasdb.jdbc.config.JdbcDataSourceConfiguration
```

Comment:
No, since the point of this PR is to remove the dropwizard dependency. This is how it is done in atlasdb-config as well.